.github/dependabot.yml

```diff
@@ -0,0 +1,23 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/en/code-security/dependabot/working-with-dependabot/dependabot-options-reference
+
+version: 2
+
+# this would enable querying all packages from our repo first,
```

Comment:
Can you please remove the commented out JFrog stuff? Maven isn’t relevant here (at least in the main branch …)

Reply:
sure, done